root/components/EntityTabs.js

```diff
@@ -133,7 +133,10 @@ function buildLinks(
     links.push(buildLink(l('Tags'), entity, 'tags', page));
   }
 
-  if (entityProperties.ratings || entityProperties.reviews) {
+  if (
+    (/*:: entityProperties.ratings != null && */ entityProperties.ratings) ||
+    entityProperties.reviews
+  ) {
```

Comment:
Flow didn't like `reviews` being accessed here without checking that `ratings` isn't null first (not exactly sure why).

Reply:
Did it also complain if you assigned `entityProperties.ratings || entityProperties.reviews` to a constant?

Reply:
That doesn't help because the error is on the `entityProperties.reviews` access (so it's just moving the location of the error).  I decided to mark it with `$FlowIssue` instead though, since that should at least tell us if it's fixed.